### ISSUE TITLE
petab_import requires non mandatory petab columns

### ISSUE THREAD

**Issue by MerktSimon** [ICB-DCM/AMICI#973]
measurement table has to have preequilibration, noiseParameter and observedParameter column which are not mandatory in Petab

**Comment by dweindl** [comment]
Yes, should be optional. Please provide stack traces, to make it easier to fix.

If `noiseParameters` is empty, sigma should be set to 1.0 during import.

**Comment by MerktSimon** [comment]
stack traces?

**Comment by dweindl** [comment]
> stack traces?

Something like that: 
https://github.com/ICB-DCM/AMICI/issues/965#issuecomment-589328582

**Comment by MerktSimon** [comment]
```
KeyError                                  Traceback (most recent call last)

<ipython-input-3-18c90a2228a6> in <module>
     15 sim_results = simulate_petab(petab_problem=petab_problem,
     16                              amici_model=amici_model,
---> 17                              solver=amici_solver)
     18 

~/git/AMICI/python/sdist/amici/logging.py in wrapper_timer(*args, **kwargs)
    142         def wrapper_timer(*args, **kwargs):
    143             tstart = time.perf_counter()
--> 144             rval = func(*args, **kwargs)
    145             tend = time.perf_counter()
    146             spacers = ' ' * max(55 - len(description) - len(logger.name), 0)

~/git/AMICI/python/sdist/amici/petab_objective.py in simulate_petab(petab_problem, amici_model, solver, problem_parameters, simulation_conditions, parameter_mapping, parameter_scale_mapping, scaled_parameters, log_level)
    115         parameter_mapping=parameter_mapping,
    116         parameter_scale_mapping=parameter_scale_mapping,
--> 117         scaled_parameters=scaled_parameters)
    118 
    119     # Simulate

~/git/AMICI/python/sdist/amici/petab_objective.py in edatas_from_petab(model, petab_problem, problem_parameters, simulation_conditions, parameter_mapping, parameter_scale_mapping, scaled_parameters)
    226             parameter_mapping=cur_parameter_mapping,
    227             parameter_scale_mapping=cur_parameter_scale_mapping,
--> 228             scaled_parameters=scaled_parameters
    229         )
    230         edatas.append(edata)

~/git/AMICI/python/sdist/amici/petab_objective.py in get_edata_for_condition(condition, problem_parameters, amici_model, petab_problem, observable_ids, parameter_mapping, parameter_scale_mapping, scaled_parameters)
    441             except ValueError:
    442                 continue
--> 443             if condition[PREEQUILIBRATION_CONDITION_ID]:
    444                 condition_id = condition[PREEQUILIBRATION_CONDITION_ID]
    445             else:

~/PycharmProjects/amici-runs/venv/lib/python3.6/site-packages/pandas/core/series.py in __getitem__(self, key)
    869         key = com.apply_if_callable(key, self)
    870         try:
--> 871             result = self.index.get_value(self, key)
    872 
    873             if not is_scalar(result):

~/PycharmProjects/amici-runs/venv/lib/python3.6/site-packages/pandas/core/indexes/base.py in get_value(self, series, key)
   4403         k = self._convert_scalar_indexer(k, kind="getitem")
   4404         try:
-> 4405             return self._engine.get_value(s, k, tz=getattr(series.dtype, "tz", None))
   4406         except KeyError as e1:
   4407             if len(self) > 0 and (self.holds_integer() or self.is_boolean()):

pandas/_libs/index.pyx in pandas._libs.index.IndexEngine.get_value()

pandas/_libs/index.pyx in pandas._libs.index.IndexEngine.get_value()

pandas/_libs/index.pyx in pandas._libs.index.IndexEngine.get_loc()

pandas/_libs/hashtable_class_helper.pxi in pandas._libs.hashtable.PyObjectHashTable.get_item()

pandas/_libs/hashtable_class_helper.pxi in pandas._libs.hashtable.PyObjectHashTable.get_item()

KeyError: 'preequilibrationConditionId'

```

**Comment by dweindl** [comment]
Fixed in v0.10.20
